pkg/model/bundle.go

```diff
@@ -64,12 +65,76 @@ func (c *Contributor) String() string {
 	return fmt.Sprintf("%s <%s>", c.Name, c.Email)
 }
 
+const (
+	ConsumableStorePathTypeDescriptor = iota
+	ConsumableStorePathTypeFileList
+)
+
+type ConsumableStorePathMetadata struct {
+	Type     byte
+	BundleID string
+	Index    uint64
+}
+
+func GetConsumableStorePathMetadata(path string) (ConsumableStorePathMetadata, error) {
```

Comment:
Please add a line as to what this method is expected to do.

Reply:
have added